app/assets/javascripts/cable.js

```diff
@@ -11,3 +11,5 @@
   App.cable = ActionCable.createConsumer();
 
 }).call(this);
+
+const f = 19;
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
'f' is assigned a value but never used                                 no-unused-vars